src/agent/Cargo.toml

```diff
@@ -72,9 +72,7 @@ clap = { version = "3.0.1", features = ["derive"] }
 openssl = { version = "0.10.38", features = ["vendored"] }
 
 # Image pull/decrypt
-image-rs = { git = "https://github.com/confidential-containers/guest-components", rev = "53ddd632424432077e95d3901deb64727be0b4c1", default-features = false, features = [
-    "kata-cc-native-tls",
-] }
+image-rs = { git = "https://github.com/confidential-containers/guest-components", rev = "5df1101", default-features = false, features = ["kata-cc-native-tls","verity"] }
```

Comment:
Hey @ChengyuZhu6 `53ddd6324` is ahead of `5df1101` so would be a downgrade, so can you reverse the revision update of this please? Thanks

Reply:
Sure!

Reply:
Fixed.